components/Assessment/HackerEntry.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import styled from 'styled-components';
+import { ASSESSMENT_COLOR, MAX_SCORE, COLOR } from '../../constants';
+
+const HackerName = styled.span`
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 19px;
+  color: ${COLOR.MIDNIGHT_PURPLE};
+  font-family: HK Grotesk;
+  font-style: normal;
```

Comment:
I don't think we need these styles since the `font-family` is already set globally and `font-style` is `normal` by default